CHANGELOG.md

```diff
@@ -4,6 +4,15 @@
 
 This version adds a dependency on Swift.
 
+### Breaking Changes
+
+- Make public APIs Swift friendly (#2416)
+    - Renamed `SentrySDK.addBreadcrumb(crumb:)` to `SentrySDK.addBreadcrumb(_ crumb:)`
+    - Renamed `SentryClient.add(_ crumb:)` to `SentryClient.addBreadcrumb(_ crumb:)`
+    - Renamed `SentryClient.add(_ attachment:)` to `SentryClient.addAttachment(_ attachment:)`
```

Comment:
`m`: Does two methods especially `add(_ crumb:)` is going to break almost every user. Can we keep the old method and mark it as deprecated, please?